flang/test/Lower/intrinsic-procedures/command_argument_count.f90

```diff
@@ -0,0 +1,10 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
Minor nit: I think bbc has an option to set the size of the default integer kind, so you could add another RUN line with 64-bit default integer and check for the cast too.

Reply:
So I tried to set the default kind with bbc using the "--default-kinds=<default-kind-string>" argument but I couldn't get it to work - I also couldn't see it being used anywhere. @banach-space suggested that it might be a dead argument, and suggested swapping to using the flang-new driver with "-fdefault-integer-8" argument instead.

Reply:
Yes, bbc doesn't support changing the default kinds.

It is possible to write tests in FIR, which can be run via tco or fir-opt, that change the default kinds since they are encoded in a string attribute.